Remove -liberty from Makefiles.  Symtab doesn't really use this and it
causes build failures on systems that don't have it.

diff -Naurb symtabAPI.orig/src/core/symtabAPI/i386-unknown-linux2.4/Makefile symtabAPI/src/core/symtabAPI/i386-unknown-linux2.4/Makefile
--- symtabAPI.orig/src/core/symtabAPI/i386-unknown-linux2.4/Makefile	2008-01-23 14:25:07.000000000 -0600
+++ symtabAPI/src/core/symtabAPI/i386-unknown-linux2.4/Makefile	2011-01-27 16:08:53.000000000 -0600
@@ -64,7 +64,7 @@
 endif
 LIBS    += -lgcc
 
-LIBS	+= -liberty
+# LIBS	+= -liberty
 
 AS		= /usr/bin/as
 
diff -Naurb symtabAPI.orig/src/core/symtabAPI/ppc32_linux/Makefile symtabAPI/src/core/symtabAPI/ppc32_linux/Makefile
--- symtabAPI.orig/src/core/symtabAPI/ppc32_linux/Makefile	2008-01-23 14:24:03.000000000 -0600
+++ symtabAPI/src/core/symtabAPI/ppc32_linux/Makefile	2011-01-27 16:02:24.000000000 -0600
@@ -65,7 +65,7 @@
 endif
 LIBS    += -lgcc
 
-LIBS	+= -liberty
+# LIBS	+= -liberty
 
 AS		= /usr/bin/as
 
diff -Naurb symtabAPI.orig/src/core/symtabAPI/ppc64_linux/Makefile symtabAPI/src/core/symtabAPI/ppc64_linux/Makefile
--- symtabAPI.orig/src/core/symtabAPI/ppc64_linux/Makefile	2008-01-17 14:03:38.000000000 -0600
+++ symtabAPI/src/core/symtabAPI/ppc64_linux/Makefile	2011-01-27 16:14:22.000000000 -0600
@@ -66,7 +66,7 @@
 endif
 LIBS    += -lgcc
 
-LIBS	+= -liberty
+# LIBS	+= -liberty
 
 AS		= /usr/bin/as
 
